requirements.txt

```diff
@@ -1,3 +1,3 @@
-Django>=1.8,<2.0
```

Comment:
Ah! This is the changed that confused circle-ci. It first tries to install requirements.txt then move on to tox.


I'll try today to create a circle.yml that skips this step. We usually have such a file in our project. Otherwise, we'll just rename requirements.txt to requirements-dev.txt (but I prefer to try the circle.yml route first).